Resources/Prototypes/_White/EntityLists/pmm_weapons.yml

```
@@ -0,0 +1,26 @@
+    - WeaponShotgunSawn
+    - WeaponRevolverDeckard
+    - WeaponRevolverInspector
+    - WeaponRevolverPython
+    - WeaponRevolverMateba
+    - WeaponLightMachineGunL6
+    - WeaponPistolViper
+    - WeaponPistolCobra
+    - WeaponPistolMk58
+    - WeaponPistolN1984
+    - WeaponSniperMosin
+    - WeaponSniperHristov
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)